setup.py

```diff
@@ -89,6 +89,7 @@ def run(self):
     scripts=[
         "bin/baseplate-serve{:d}".format(sys.version_info.major),
         "bin/baseplate-script{:d}".format(sys.version_info.major),
+        "bin/tshell",
```

Comment:
Do I need a `tshell2` and `tshell3` similar to the serve and script commands?

Reply:
Yeah, I think that makes most sense for consistency.

Should this be `baseplate-` prefixed? It's almost a generic thrift shell but it does tie into baseplate's span stuff (as it should) so it's definitely custom.

Reply:
> Should this be `baseplate-` prefixed?

Sounds reasonable. It also doesn't have to be called `tshell` if you have some other name that you would prefer.